packages/edit-post/src/store/actions.js

```diff
@@ -194,6 +194,19 @@ export const toggleFeature =
 			.dispatch( preferencesStore )
 			.toggle( 'core/edit-post', feature );
 
+/**
+ * Triggers an action used to set a feature flag to a desired state.
+ *
+ * @param {string}  feature Feature name.
+ * @param {boolean} value   Feature value
+ */
+export const setFeature =
+	( feature, value ) =>
+	( { registry } ) =>
+		registry
+			.dispatch( preferencesStore )
+			.set( 'core/edit-post', feature, value );
```

Comment:
It'd be good to avoid merging this as a stable API. I plan to deprecate the function above this (`toggleFeature`) in the future in favour of using the preferences API directly.

It should be relatively easy to replace this with:
```js
const { set: setPreference } = useDispatch( preferencesStore );

 // ...
 
 setPreference( 'core/edit-post', 'fixedToolbar', 'false' );
 ```